python_apps/media-monitor/install/sysvinit/airtime-media-monitor

```diff
@@ -9,12 +9,26 @@
 # Short-Description: Manage airtime-media-monitor daemon
 ### END INIT INFO
 
-USERID=WEB_USER
-GROUPID=WEB_USER
-NAME=Airtime\ Media\ Monitor
+USERID=www-data
+GROUPID=www-data
+NAME=airtime-media-monitor
```

Comment:
Look how much simpler this script becomes when you don't butcher it and understand how daemonization works. I used $NAME correctly here which eliminates the hardcoding in the old version, and now that we avoid the accidental fork in pypo/media_monitor, the LSB-supplied "status_of_proc" function magically works, and the "status" function now works without any hacks like airtime-check-system or monit.